jupyterhub/templates/proxy/service.yaml

```
@@ -1,8 +1,10 @@
 {{- $HTTPS := .Values.proxy.https.enabled -}}
-{{- $autoHTTPS := and $HTTPS (and (eq .Values.proxy.https.type "letsencrypt") .Values.proxy.https.hosts) -}}
-{{- $offloadHTTPS := and $HTTPS (eq .Values.proxy.https.type "offload") -}}
+# Automatic HTTPS via Let's Encrypt requires list of hostnames to be turned on
+{{- $autoHTTPS := and $HTTPS and (eq .Values.proxy.https.type "letsencrypt") .Values.proxy.https.hosts) -}}
```

Syntax error due to a mismatch of parenthesis.

```suggestion
{{- $autoHTTPS := and $HTTPS (and (eq .Values.proxy.https.type "letsencrypt") .Values.proxy.https.hosts) -}}
```